sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -1,5 +1,20 @@
 namespace Azure
 {
+    public partial class CloudMachine
+    {
+        public CloudMachine(Microsoft.Extensions.Configuration.IConfiguration configuration) { }
+        public CloudMachine(System.IO.Stream configurationContent) { }
+        public CloudMachine(string configurationFile = ".\\cloudconfig.json") { }
+        public string DisplayName { get { throw null; } set { } }
+        public string Id { get { throw null; } }
+        public string Region { get { throw null; } }
+        public string SubscriptionId { get { throw null; } }
+        public static Azure.CloudMachine Create(string subscriptionId, string region) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public void Save(System.IO.Stream stream) { }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
```

Comment:
Curious: Why EBN these?

Reply:
These are to be used by tools only/mainly. I don't actually want the average users to ever call these.